src/lib/Runner.ts

```diff
@@ -49,14 +49,17 @@ export class Runner {
     return this.queue.getRunning();
   };
 
+  getWaitingAndQueued = async () => {
+    const queued = await this.getQueue();
+    const waiting = await this.queue.getStatusesForWaitingRequests();
+    return [...queued, ...waiting];
+  };
+
   moveFromQueueToRunning = async (landRequestStatus: LandRequestStatus, lockId: Date) => {
     const landRequest = landRequestStatus.request;
     const running = await this.getRunning();
     const runningTargetingSameBranch = running.filter(
-      build =>
-        build.request.pullRequest.targetBranch === landRequest.pullRequest.targetBranch &&
```

Comment:
So, we've removed this, but it's not taken care of by `isAllowedToMerge` is that correct?